src/gossip/net.zig

```diff
@@ -16,9 +16,8 @@ pub const SocketAddr = union(enum(u8)) {
         },
     };
 
-    pub fn parse(bytes: []const u8) !Self {
-        // TODO: parse v6 if v4 fails
-        return parse_v4(bytes);
+    pub fn parse(bytes: []const u8, ip_port: u16) !Self {
```

Comment:
rn this breaks the `zig build run` cmd (see `cmd.zig`) - can you parse ipv6 assuming "[{ipv6_ip}]:port" format?

Reply:
What if we edit cmd to be compatible with this change?

Reply:
true that would not be super elegant, I will add some parsing logic to split the post from the ip